app/src/main/res/values/strings.xml

```diff
@@ -78,6 +78,8 @@
     <string name="profile_picture">Profile Picture</string>
     <string name="system_language">System Language</string>
     <string name="system_region">System Region</string>
+    <string name="internet_enabled">The system cannot use internet</string>
+    <string name="internet_disabled">The system can use internet</string>
```

Comment:
Honestly you should invert these strings, as you also use them inverted in the preference xml.

Reply:
Just use a single preference string tbh instead of having it change. Would be clearer

Reply:
changed to single string